packages/loot-core/src/server/budget/goaltemplates.ts

```diff
@@ -521,13 +534,13 @@ async function applyCategoryTemplate(
           target = 0;
           remainder = Math.abs(remainder);
         }
-        let diff = num_months > 0 ? Math.round(target / num_months) : 0;
+        let diff = num_months > -1 ? Math.round(target / (num_months + 1)) : 0;
         if (num_months < 0) {
           errors.push(
             `Non-repeating schedule ${template.name} was due on ${next_date_string}, which is in the past.`,
           );
           return { errors };
-        } else if (num_months > 0) {
+        } else if (num_months > -1) {
```

Comment:
Can you switch these lines across the file to `>= 0`? That reads more naturally to me and should be equivalent (right?).

Reply:
Agreed.  I'll make that change.